### DISCUSSION TITLE
SBP Logos

### DISCUSSION THREAD

**Discussion by SecurityBestPractices** [Security-Best-Practices/SBP-web#4] [Ideas]
Logos:

**Comment by SecurityBestPractices** [comment]
Avatar 1
![image](https://github.com/user-attachments/assets/6b3661e2-7f4d-41cf-9dfe-0151edb2099f)

**Comment by SecurityBestPractices** [comment]
Avatar 2
![image](https://github.com/user-attachments/assets/b4f256a0-1210-49cf-a618-9de20502cab3)

**Comment by SecurityBestPractices** [comment]
![Logo_01a](https://github.com/user-attachments/assets/baa8c57f-a301-449e-9664-3d810ef4a857)

**Comment by SecurityBestPractices** [comment]
![Logo_01b](https://github.com/user-attachments/assets/b6b0b1cf-8955-49bb-8366-767b608aef2b)

**Comment by SecurityBestPractices** [comment]
![Logo_01c](https://github.com/user-attachments/assets/2495ffe0-9098-4ea6-beb5-bb0354463996)

**Comment by SecurityBestPractices** [comment]
![Logo_02a_HEADER](https://github.com/user-attachments/assets/6c050d40-5d9f-436a-a9c5-369495e48ae2)

**Reply by stratofax** [reply]
I think this is the best choice for the menu bar, and we can use this as the basis for your color palette

**Comment by SecurityBestPractices** [comment]
![Logo_02b](https://github.com/user-attachments/assets/135a6971-c7c3-407e-af53-32ca73392ccd)

**Comment by SecurityBestPractices** [comment]
![Logo_02d](https://github.com/user-attachments/assets/c77cf57d-6002-4045-ad77-8d47a97af387)
